Stack/C++/Stack using LL.cpp

```diff
@@ -0,0 +1,159 @@
+#include<iostream>
+using namespace std;
+
+class node
+{

```

Comment:
Please change the files name with camelcase

Reply:
> Please change the files name with camelcase

Done